ext/opcache/zend_accelerator_module.c

```diff
@@ -491,6 +494,26 @@ void zend_accel_info(ZEND_MODULE_INFO_FUNC_ARGS)
 			php_info_print_table_row(2, "Hash keys restarts", buf);
 			snprintf(buf, sizeof(buf), ZEND_ULONG_FMT, ZCSG(manual_restarts));
 			php_info_print_table_row(2, "Manual restarts", buf);
+
+			start_time = php_format_date("d-M-Y H:i:s e", sizeof("d-M-Y H:i:s e")-1, ZCSG(start_time), 1);
```

Comment:
I suggest to change this to ISO 8601, at least for the date format (i.e. `Y-m-d`).

Reply:
I actually was trying to be consistent with how PHP formats error logs:
https://github.com/php/php-src/blob/master/main/main.c#L836
https://github.com/php/php-src/blob/master/ext/standard/mail.c#L416

But sure, I can change this.

Reply:
`Y-m-d` everywhere would be great!

Reply:
Not a problem at all, updated